examples/trl_mixin/ex_trl_constant.py

```diff
@@ -39,21 +39,23 @@ def formatting_prompts_func(example):
 response_template = "Answer:"
 collator = DataCollatorForCompletionOnlyLM(response_template, tokenizer=tokenizer)
 
-training_args = TrainingArguments(
+training_args = dict(
     output_dir=output_dir,
-    num_train_epochs=0.6,
+    num_train_epochs=0.1,
```

Comment:
I assume 0.6 is a good number to demonstrate for the example, is there a reason why it should be changed?

Reply:
Sorry I changed it back